### PR TITLE
convert to Db2 on Cloud Lite

### PR DESCRIPTION
**Pull request by ibmrcruicks**
Db2WH offers no free plan anymore, but Db2 (dashdb for Transactions) does.

updated all Bluemix links for current equivalents

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -2,16 +2,16 @@
 declared-services:
   dashDB-nodesample:
     label: dashDB
```
Comment (prsahoo) [inline manifest.yml]: I am hoping the label would need to be changed to "dashDB For Transactions"

**Review by prsahoo — CHANGES_REQUESTED** [review]
Hi Ross, Thanks for submitting the PR.  I am hoping we would suggested change in the manifest.yml file.  Can you please make that change and then we can merge it.  Thanks.

File: manifest.yml (hunk 1 above)
Comment (ibmrcruicks) [reply manifest.yml]: good point! updated

Comment (prsahoo) [reply manifest.yml]: Thank you.

**Pull request merged**